downstream/assemblies/eda/assembly-eda-credential-types.adoc

```diff
@@ -0,0 +1,22 @@
+[id="eda-credential-types"]
+
+= Credential types
+
+Five built-in credential types are supported for {EDAcontroller}:
```

Comment:
Note to Reviewers: This statement will be updated later today to state "that we ship with a few Credential types for use in Project Sync, Controller, Registry and Event Streams and these credential types cannot be deleted or edited by the user," per discussion with Madhu Kanoor.